src/odemis/acq/move.py

```diff
@@ -2871,31 +2885,29 @@ def moveAbs(self, pos: Dict[str, float], **kwargs) -> Future:
                 pos[key] = self.position.value[key]
 
         # pos is a position, so absolute conversion
-        pos_stage = self.pm.from_sample_stage_to_stage_position(pos)
+        pos_stage = self._pm.from_sample_stage_to_stage_position(pos)
         logging.debug("converted absolute move from %s to %s", pos, pos_stage)
         return self._stage_bare.moveAbs(pos_stage, **kwargs)
 
     @isasync
     def moveRelChamberReferential(self, shift: Dict[str, float]) -> Future:
-        """Move the stage vertically in the chamber. This is non-blocking.
-        From OpenFIBSEM.
+        """Move the stage vertically in the chamber. This is non-blocking. From OpenFIBSEM.
         The desired input shift (x, z) is transformed to x, y, z axis components such that the
-        the stage moves in the vertical direction in the chamber. For TFS, the z-axis is
-        attached to the tilt, so the tilt angle must be taken into account.
-        For Tescan systems, the z-axis is always vertical, so this function is not required.
+        the stage moves in the vertical direction in the chamber.
+        The input shift is expected in the sample-stage coordinates. A feature in the FIB FoV is
+        expected to be moved by dx in the x-image coordinates, and dz in the y-image coordinates. The
+        feature will not be moved in the SEM FoV. To achieve this, we compute the stage-bare movement such
+        that the stage moves vertically in the chamber, resulting in the desired behavior in the SEM/FIB FoV.
```

Comment:
Ok. Thanks for clarifying. I think this is now precise enough. Just for me, in practice for now, this function is only used to align the coincidental point for SEM/FIB, when in SEM imaging posture, right? Or also maybe in FIB imaging? Never in FLM posture, or even MILLING, right?

Reply:
yes just used to align coincidence of sem/fib, and will work at any posture (SEM, FIB, MILL). it will do the same thing at FM, but theres no coincidence for meteor so in practice you'd never use it there.